cpp/lazperf/vlr.cpp

```diff
@@ -433,5 +433,144 @@ vlr_header copc_vlr::header() const
     return vlr_header { 0, "entwine", 1, (uint16_t)size(), "COPC offsets" };
 }
 
+
+copc_extents_vlr::copc_extents_vlr()
+{}
+
+
+copc_extents_vlr::copc_extents_vlr(int byteSize)
+{
+    int itemCount = byteSize / sizeof(CopcExtent);
+    items.resize(itemCount);
+}
+
+
+copc_extents_vlr::~copc_extents_vlr()
+{}
+
+
+copc_extents_vlr::CopcExtent::CopcExtent() :
```

Comment:
This seems unnecessary. Why not just provide a function like "add(min, max)" and the data can go into a pair.